### PR TITLE
add sweetbread recipe

### PR DESCRIPTION
**Pull request by tsulh**
Adds the most common way of turning raw offals into delicious sweetbreads (instead of sickening cooked offals).

### CODE REVIEW

File: data/json/recipes/recipe_food.json [hunk 1]
```diff
@@ -134,6 +134,64 @@
...
+  "subcategory": "CSC_FOOD_MEAT",
+  "difficulty": 4,
+  "skill_used": "cooking",
+  "time": 40000,
+  "autolearn": true,
+  "batch_time_factors":[67, 5],
+  "qualities":[
+    {"id":"CUT","level":1,"amount":1},
+    {"id":"BOIL","level":2,"amount":1},
+    {"id":"COOK","level":2,"amount":1}
```
Comment (Coolthulhu) [inline data/json/recipes/recipe_food.json]: "amount":1 is redundant and should be removed

**Comment by tsulh** [comment]
I've changed the commit (introduced both recommended changes).

**Comment by tsulh** [comment]
I'm not sure whether +5 fun is not too little, because cooking oil and vinegar both seem to be expensive ingridients, and the meal is considered tasty irl. But I've played less than 90 cata-days , so it might be the start-game impression. Is there a reason why it shouldn't be higher than 5?

**Comment by Rivet-the-Zombie** [comment]
>Is there a reason why it shouldn't be higher than 5?
It's very easy to get high fun from meat pies, and they're much simplier to cook.

Meat pies' morale is likely a bit on the high side.

**Comment by tsulh** [comment]
Pelmeni give +6 for each portion, though it's quite a simple and practical food (just flour and meat, the same as the pie). What about setting morale of the sweetbread to +8 or +10?

**Comment by Coolthulhu** [comment]
> cooking oil and vinegar both seem to be expensive ingridients

Your recipe doesn't use up any cooking oil - it will work with just 1 of it forever.
Vinegar isn't very expensive, as it spawns in gallon jugs and doesn't have many uses apart from making batteries.

> Is there a reason why it shouldn't be higher than 5?

Well OK, maybe a bit more than 5 could still be OK.
But don't go over 10. Most things above 10 fun are either addicting or uncraftable. Rare exceptions (deluxe sandwich, hosomaki, lasagne) take hard to get components.
Sweetbread is cheaper than pizza, which is 10 fun. The only component which is hard to store or get on the list is offal, which has no good use at the moment. Pizza is also booklearn, unlike sweetbread (although it is worth 1-2 points at most).

File: data/json/items/comestibles.json [hunk 2]
```diff
@@ -10021,6 +10021,27 @@
...
+        "name_plural" : "sweetbreads",
+        "weight" : 150,
+        "color" : "brown",
+        "spoils_in" : 48,
+        "comestible_type" : "FOOD",
+        "symbol" : "%",
+        "quench" : 1,
+        "healthy" : 1,
+        "nutrition" : 24,
+        "description" : "Delicious and tender organ meats. First boiled, then breaded and fried. They have interesting flavor and unparalleled texture.",
```
Comment (Coolthulhu) [inline data/json/items/comestibles.json]: Should be double spaces after periods.

**Comment by tsulh** [comment]
Ok, fixed the spaces and changed fun to 8.
Infinitely reusable cooking oil doesn't seem nice, but I guess it's possible to filter it through rag or something... and the filtering is too tedious to be introduced into the game.

**Comment by dodzy** [comment]
some of the oil would go into the bread though, or coat it.

**Comment by Rivet-the-Zombie** [comment]
[It's not actually bread.](https://en.wikipedia.org/wiki/Sweetbread)

**Pull request merged**